open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -23,9 +23,11 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.        def test_column_to_show_by_getting_column_to_diff_with_none_column_to_diff(
</pre><p>Did you run all your code? Are you sure this test case will work? You are not testing column_to_diff is None. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Sorry, Confused, Done</p>